spec/classes/init_spec.rb

```diff
@@ -226,60 +226,60 @@
     let(:mandatory_params) { {} }
 
     validations = {
-      'absolute_path' => {
+      'Stdlib::Absolutepath' => {
         :name    => %w(config_path),
         :valid   => ['/absolute/filepath', '/absolute/directory/'],
-        :invalid => ['../invalid', %w(array), { 'ha' => 'sh' }, 3, 2.42, true, nil],
-        :message => 'expects.*Variant\[Stdlib::Windowspath.*Stdlib::Unixpath',
+        :invalid => ['../invalid', %w(array), { 'ha' => 'sh' }, 3, 2.42, false, nil],
+        :message => 'expects a (match for|match for Stdlib::Absolutepath =|Stdlib::Absolutepath =) Variant\[Stdlib::Windowspath.*Stdlib::Unixpath', # Puppet (4.x|5.0 & 5.1|5.x)
```

Comment:
They did it again, changing error messages for minor versions.
Puppet 5.0 & 5.1 error messages are different from >= 5.2.... tzzzz....